cmd/jaeger/config-cassandra.yaml

```diff
@@ -20,14 +20,24 @@ extensions:
     backends:
       some_storage:
         cassandra:
-          keyspace: "jaeger_v1_dc1"
-          username: "cassandra"
-          password: "cassandra"
+          schema: 
+            keyspace: "jaeger_v1_dc1"
+          connection:
+            auth: 
```

Comment:
Q: what are other authenticators supported by gocql? Does the u/pwd pair apply to all of them? If not then it probably shouldn't be squashed , rather nested under basic.